tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -806,6 +806,21 @@ class ExecutionTest {
 
     VERIFY_SUCCEEDED(CreateDXGIFactory1(IID_PPV_ARGS(&factory)));
     if (GetTestParamUseWARP(UseWarpByDefault())) {
+
+      // The WARP_DLL runtime parameter can be used to specify a specific DLL to
+      // load.  To force this to be used, we make sure that this DLL is loaded
+      // before attempting to create the device.
+      HMODULE ExplicitlyLoadedWarpDll = NULL;
+      WEX::Common::String WarpDllPath;
+      if (SUCCEEDED(WEX::TestExecution::RuntimeParameters::TryGetValue(
+              L"WARP_DLL", WarpDllPath))) {
+        WEX::Logging::Log::Comment(WEX::Common::String().Format(
+            L"WARP_DLL requested: %ls", (const wchar_t *)WarpDllPath));
+        ExplicitlyLoadedWarpDll = LoadLibraryExW(WarpDllPath, NULL, 0);
```

Comment:
>NULL [](http://example.com/codeflow?start=62&length=4)

should 'nullptr' be used for new code?

Reply:
For "proper" pointers I would, but I try and use NULL when interacting with Win32 API's that explicitly state "NULL" as having a meaning to the API.